src/uu/mknod/src/mknod.rs

```diff
@@ -140,28 +193,37 @@ pub fn uu_app() -> Command {
                 .value_hint(clap::ValueHint::AnyPath),
         )
         .arg(
-            Arg::new("type")
+            Arg::new(options::TYPE)
                 .value_name("TYPE")
                 .help("type of the new file (b, c, u or p)")
                 .required(true)
                 .value_parser(parse_type),
         )
         .arg(
-            Arg::new("major")
-                .value_name("MAJOR")
+            Arg::new(options::MAJOR)
+                .value_name(options::MAJOR)
                 .help("major file type")
                 .value_parser(value_parser!(u64)),
         )
         .arg(
-            Arg::new("minor")
-                .value_name("MINOR")
+            Arg::new(options::MINOR)
+                .value_name(options::MINOR)
                 .help("minor file type")
                 .value_parser(value_parser!(u64)),
         )
+        .arg(
+            Arg::new(options::SELINUX)
+                .short('Z')
+                .help("set SELinux security context of each created directory to the default type")
+                .action(ArgAction::SetTrue),
+        )
+        .arg(Arg::new(options::CONTEXT).long(options::CONTEXT).value_name("CTX").help(
+            "like -Z, or if CTX is specified then set the SELinux or SMACK security context to CTX",
+        ))
```

Comment:
GNU `mknod` allows to use `--context` without a value. I think you can define it with clap with a [`default_missing_value`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.default_missing_value).